backend/internal/model/game.go

```diff
@@ -145,3 +154,34 @@ func (g *game) GetShotsData() ShotsData {
 
 	return g.shotsData
 }
+
+func (g *game) UpdateManualGoals(teamID int, action string) error {
+	g.m.Lock()
+	defer g.m.Unlock()
+
+	switch action {
+	case "add":
+		switch teamID {
+		case config.TeamWhite:
+			g.manualGoals.AddedWhite++
+		case config.TeamBlue:
+			g.manualGoals.AddedBlue++
+		default:
+			return errors.New("bad team ID")
+		}
+		return nil
+	case "sub":
+		switch teamID {
+		case config.TeamWhite:
+			g.manualGoals.SubtractedWhite++
+		case config.TeamBlue:
+			g.manualGoals.SubtractedBlue++
+		default:
+			return errors.New("bad team ID")
+		}
+		return nil
```

Comment:
It would never return nil, because a few lines above there is the `default` statement

```suggestion
```

Reply:
The default case won't be executed if function arguments are valid. Then the function returns nil. We have checked it in our [tests](https://github.com/HackYourCareer/SmartKickers/pull/100/files#diff-e0bdbdb4238f2199e4b6e95c7bb3e24adcb309c41900f36a00f8aebe3c681266R153-R156)

Reply:
that's right 👍